release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3191,4 +3191,30 @@ link:https://access.redhat.com/solutions/6970049[{product-title} 4.10.25 contain
 [id="ocp-4-10-25-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-26"]
+=== RHSA-2022:5875 - {product-title} 4.10.26 bug fix and security update
+
+Issued: 2022-08-08
+
+{product-title} release 4.10.26, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:5875[RHSA-2022:5875] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5874[RHBA-2022:5874] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6971191[{product-title} 4.10.26 container image list]
+
+[id="ocp-4-10-26-bug-fixes"]
+==== Bug fixes
+
+* Previously, new regions were not recognized by the AWS SDK, which meant the machine controller could not use them. This was due to AWS SDK only knowing regions from the time it was vendored. With this update, administrators can use `DescribeRegions` to check the specified region for a machine and create them in regions unknown to SDK. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2109124[*BZ#2109124*])
```

Comment:
Suggestion to be more concise: "Previously, new regions were not recognized by the AWS SDK and the machine controller could not use them."

"This was due to" - This sentence is a little awkward. Maybe rewrite to something like "This problem occurred because the AWS SDK only recognized regions from the time it was vendored." Also not sure if there is a better word than vendored?

Reply:
Also, not clear to me which component _it_ is. The AWS SDK or the region?